src/auth/AuthProvider.tsx

```diff
@@ -99,15 +104,24 @@ class AuthProvider extends React.Component<
     const token = getAuthToken();
     if (!!token && !user) {
       this.verifyToken(token);
+    } else {
+      if (isCredentialsManagementAPISupported) {
```

Comment:
Wouldn't it make more sense to move this `if` inside the `loginWithCredentialsManagementAPI` function?

Reply:
I think it makes more sense to check it here since we're about to call an API which we don't know if it's supported by user's browser. But it's purely semantic difference and I'll change it if you want 🤷‍♂